src/transformers/processing_utils.py

```diff
@@ -1076,12 +1076,30 @@ def apply_chat_template(
         conversation: Union[List[Dict[str, str]]],
         chat_template: Optional[str] = None,
         tokenize: bool = False,
+        return_dict: bool = False,
+        num_frames: int = None,
+        video_load_backend: str = "opencv",
+        processor_kwargs: Dict = {},
```

Comment:
some of these should be `video_kwargs` IMO, with unpack typing no?

Reply:
you mean `num_frames` and `backend`? I think it's better this way and also I had an idea to force users pass `frames` in the conversation dict, which imo makes sense if one wants different number of frames within one conversation. Currently we cannot batch an input with different frame number afaik but might be a feature for future VLMs

So like this `{type: video, video_url: https://..., num_framees: 32}`. WDYT, is this better than passing as kwargs?